app/controllers/openid_connect/user_info_controller.rb

```diff
@@ -0,0 +1,38 @@
+module OpenidConnect
+  class UserInfoController < ApplicationController
+    skip_before_action :verify_authenticity_token
+    before_action :authenticate_identity_via_bearer_token
+
+    attr_reader :current_identity
+
+    def show
+      render json: OpenidConnectUserInfoPresenter.new(current_identity).user_info
+    end
+
+    private
+
+    def authenticate_identity_via_bearer_token
```

Comment:
Nice. Much better! What do you think about passing in the specific `HTTP_AUTHORIZATION` header to `IdTokenVerifier` since that's what it's validating? It doesn't need the entire `request.env` object as far as I can tell.